contrib/vspec2ttl/vspec2ttl.py

```diff
@@ -55,168 +55,263 @@ def usage():
+    g.add((ontology, OWL.imports, URIRef("http://purl.oclc.org/NET/ssnx/qu/qu-rec20")))
+
     belongsTo = VssoConcepts.BELONGS_TO.uri
     g.add((belongsTo, RDF.type, OWL.ObjectProperty))
-    g.add((belongsTo,RDFS.label, Literal(VssoConcepts.BELONGS_TO.value,lang="en")))
-    g.add((belongsTo,RDFS.range, VssoConcepts.VEHICLE_COMP.uri))
-    g.add((belongsTo,RDFS.domain, VssoConcepts.VEHICLE_STAT.uri))
-    g.add((belongsTo,RDFS.domain, VssoConcepts.VEHICLE_PROP.uri))
-
-    holdsValue = VssoConcepts.HOLDS_VALUE.uri
-    g.add((holdsValue, RDF.type, OWL.DatatypeProperty))
-    g.add((holdsValue,RDFS.subPropertyOf, OWL.topDataProperty))
-    g.add((holdsValue,RDFS.domain, VssoConcepts.VEHICLE_PROP.uri))
-    g.add((holdsValue,RDFS.label, Literal(VssoConcepts.HOLDS_VALUE.value,lang="en")))
-
-    partOf = VssoConcepts.PART_OF_VEHICLE.uri
-    g.add((partOf, RDF.type, OWL.ObjectProperty))
-    g.add((partOf,RDFS.subPropertyOf, OWL.topObjectProperty))
-    g.add((partOf,RDFS.domain, VssoConcepts.VEHICLE_COMP.uri))
-    g.add((partOf,RDFS.range, URIRef(f"{VssoConcepts.EMPTY.uri_string}Vehicle")))
-    g.add((partOf,RDFS.label, Literal("partOf Vehicle",lang="en")))
-    
-    partOf = VssoConcepts.PART_OF_VEH_COMP.uri
-    g.add((partOf, RDF.type, OWL.ObjectProperty))
-    g.add((partOf,RDFS.subPropertyOf, OWL.topObjectProperty))
-    g.add((partOf,RDFS.domain, VssoConcepts.VEHICLE_COMP.uri))
-    g.add((partOf,RDFS.range, VssoConcepts.VEHICLE_COMP.uri))
-    g.add((partOf,RDFS.label, Literal("partOf Vehicle Component",lang="en")))
-
-    hasComponentInstance = VssoConcepts.HAS_COMP_INST.uri
-    g.add((hasComponentInstance, RDF.type, OWL.DatatypeProperty))
-    g.add((hasComponentInstance,RDFS.subPropertyOf, OWL.topDataProperty))
-    g.add((hasComponentInstance,RDFS.label, Literal(VssoConcepts.HAS_COMP_INST.value,lang="en")))
-
-
-    hasAttribute = VssoConcepts.HAS_ATTRIBUTE.uri
-    g.add((hasAttribute, RDF.type, OWL.ObjectProperty))
-    g.add((hasAttribute,RDFS.subPropertyOf, OWL.topObjectProperty))
-    g.add((hasAttribute, RDFS.domain, VssoConcepts.VEHICLE.uri))
-    g.add((hasAttribute, RDFS.range, VssoConcepts.VEHICLE_STAT.uri))
-    g.add((hasAttribute, RDFS.label, Literal(VssoConcepts.HAS_ATTRIBUTE.value,lang="en")))
-
-    hasDynProp = VssoConcepts.HAS_SIGNAL.uri
-    g.add((hasDynProp, RDF.type, OWL.ObjectProperty))
-    g.add((hasDynProp,RDFS.subPropertyOf, OWL.topObjectProperty))
-    g.add((hasDynProp, RDFS.domain, VssoConcepts.VEHICLE.uri))
-    g.add((hasDynProp, RDFS.range, VssoConcepts.VEHICLE_PROP.uri))
-    g.add((hasDynProp, RDFS.label, Literal(VssoConcepts.HAS_SIGNAL.value,lang="en")))
-
+    g.add((belongsTo, RDFS.subPropertyOf, OWL.topObjectProperty))
+    g.add((belongsTo, RDFS.label, Literal(VssoConcepts.BELONGS_TO.value, lang="en")))
+    g.add((belongsTo, RDFS.range, VssoConcepts.VEHICLE_COMP.uri))
+    g.add((belongsTo, RDFS.domain, VssoConcepts.VEHICLE_PROP.uri))
+
+    # holdsValue = VssoConcepts.HOLDS_VALUE.uri
```

Comment:
For commented out code I would prefer a comment explaining why it is commented (and not removed). 

(I assume that we in general do not want commented out code, but if it is just temporary it might be OK)

Reply:
Those comments means need discussion. 

We are changing the model used for generating .ttl file, the part for abstraction (i.e. the content within VSS) are ready in our proposal, but the part for describing instance (the single produced vehicle component part installed in vehicle) is not ready yet.

The commented part is originally used for instances, and now they have conflicts with our new model design. If the new model design works well, we will further adapt the commented part to fit the new design.